src/io/flutter/preview/RenderHelper.java

```diff
@@ -325,78 +317,62 @@ private void render(@NotNull RenderRequest request) {
         }
       }
 
-      // If the is no rendering server process, start a new one.
-      boolean newProcess = false;
-      if (myProcess == null) {
+      // If there is no the rendering server process, start a new one.
```

Comment:
=> If there is no rendering server process

Reply:
Fixed.